vocabularies/resource-project-lifecycle.ttl

```diff
@@ -19,7 +19,7 @@ prcls:conceptScheme a skos:ConceptScheme ;
     skos:hasTopConcept prcls:appraisal,
         prcls:development,
         prcls:exploration ;
-    skos:prefLabel "Project Class"@en .

```

Comment:
Should move Project Class to altLabel rather than delete as it is a PPDM published term, i.e.
skos:altLabel "Project Class"@en;

Reply:
Alt label added.